stable/kubernetes-dashboard/values.yaml

```diff
@@ -80,8 +80,13 @@ ingress:
   ##
   # annotations:
   #   kubernetes.io/ingress.class: nginx
-  #   nginx.ingress.kubernetes.io/secure-backends: "true"
+
   #   kubernetes.io/tls-acme: 'true'
+  ## If you use ingress-nginx < 0.21.0
```

Comment:
Maybe state to use this if you are not using the opt-in plain http backend (`enableInsecureLogin`)

Reply:
Done